measures/add_pytank/measure.rb

```diff
@@ -212,6 +227,21 @@ def run(ws, runner, usr_args)
     no.setString(1, 'Any Number')
     no.setDouble(2, num_tanks)
     ws.addObject(no)
+    
+    # handle the storage medium
+    media_lookup = {
+      'water' => 5,
+      'simplewater' => 6,
+      'pcm2x2a' => 7
+    }                
+    fluid_type = media_lookup[strg_medium] # magic number for the medium
+    # add storage medium schedule
```

Comment:
Add a schedule for the storage medium. The magic number in the variable `fluid_type` will need to match up with the enumeration value in `fluid.py`